adserver/tasks.py

```diff
@@ -759,6 +762,50 @@ def notify_on_ad_image_change(advertisement_id):
     )
 
 
+@app.task()
+def notify_of_autorenewing_flights(days_before=7):
+    """Send a note to flights set to renew automatically."""
+    # Flight must end in exactly `days_before` days
+    # to receive the notification
+    end_date = get_ad_day().date() + datetime.timedelta(days=days_before)
+
+    for flight in Flight.objects.filter(
+        live=True,
+        auto_renew=True,
+        end_date=end_date,
+    ).select_related():
+        log.info("Notifying about flight %s auto-renewing", flight)
+        if settings.FRONT_ENABLED:
+            advertiser = flight.campaign.advertiser
+            site = get_current_site(request=None)
+
+            to_addresses = [
+                u.email
+                for u in advertiser.user_set.all()
+                if u.notify_on_completed_flights
+            ]
+
+            context = {
+                "site": site,
+                "flight": flight,
+                "advertiser": advertiser,
+            }
+
+            with mail.get_connection(
+                settings.FRONT_BACKEND,
+                sender_name=f"{site.name} Flight Tracker",
+            ) as connection:
+                message = mail.EmailMessage(
+                    _("Advertising flight renewing - %(name)s") % {"name": site.name},
```

Comment:
```suggestion
                    _("Your flight will renew automatically - %(name)s") % {"name": advertiser.name},
```

Isn't EthicalAds in the from address? I'd perhaps expect this to be the company name, not our name.

Reply:
EthicalAds is in the from address but I think we want EthicalAds in the subject as well.

Compare the subjects "Your flight will renew automatically - EthicalAds" vs. "Your flight will renew automatically - $ADVERTISER". One of these tells the advertiser *where* their ads are renewing and the other one tells them that their ads are renewing and reminds them of their own company name. I think the former is better for the advertiser.

Reply:
Yea, I assume they know it's from us because of the header. I guess the main value would be for anyone managing multiple accounts on EA, and on us to make it more obvious who we're talking to in the email thread, but don't feel strongly.